tests/helpers/sharingHelper.js

```diff
@@ -0,0 +1,157 @@
+const fetch = require('sync-fetch')
+const {
+  getAuthHeaders,
+  applicationFormUrlEncoded
+} = require('../pactHelper.js')
+
+const shareEndPoint = '/ocs/v1.php/apps/files_sharing/api/v1/shares'
+const publicFilesEndPoint = '/remote.php/dav/public-files'
+
+/**
+ * returns the full and sanitized URL of the dav resource
+ *
+ * @returns {string}
+ */
+const getSharingEndPoint = function () {
+  return (process.env.PROVIDER_BASE_URL + shareEndPoint)
+    .replace(/([^:])\/{2,}/g, '$1/')
```

Comment:
same used in https://github.com/owncloud/owncloud-sdk/blob/bb47ee307afd7f0b112718fc08a48be6c0fbc805/tests/webdavHelper.js#L30

can we put it in some sanitise function used everywhere?